extensions/sql-migration/src/wizard/loginSelectorPage.ts

```diff
@@ -255,24 +276,120 @@ export class LoginSelectorPage extends MigrationWizardPage {
 			})
 			.component();
 
-		await this._loadLoginList();
+		this._windowsAuthInfoBox = this._view.modelBuilder.infoBox()
+			.withProps({
+				style: 'information',
+				text: constants.LOGIN_MIGRATIONS_SELECT_LOGINS_WINDOWS_AUTH_WARNING,
+				CSSStyles: { ...styles.BODY_CSS, 'display': 'none', }
+			}).component();
+
+		await this._createSystemLoginTablesTab(view);
+		await this._createNonSystemLoginTablesTab(view);
+
+		this._tabs = view.modelBuilder.tabbedPanel()
+			.withTabs([this._nonSystemloginTablesTab, this._systemLoginTablesTab])
+			.withProps({
+				CSSStyles: { 'margin-top': '8px', }
+			})
+			.component();
+
+		const flex = view.modelBuilder.flexContainer().withLayout({
+			flexFlow: 'column',
+			height: '100%',
+		}).withProps({
+			CSSStyles: {
+				'margin': '-20px 28px 0px 28px'
+			}
+		}).component();
+		flex.addItem(this._windowsAuthInfoBox, { flex: '0 0 auto' });
+		flex.addItem(refreshContainer, { flex: '0 0 auto' });
+		flex.addItem(this._tabs, { flex: '0 0 auto' });
+		return flex;
+	}
+
+	private async _createNonSystemLoginTablesTab(view: azdata.ModelView): Promise<void> {
+
 		this._loginCount = this._view.modelBuilder.text().withProps({
 			value: constants.LOGINS_SELECTED(
 				this.selectedLogins().length,
-				this._loginTableValues.length),
+				this._loginSelectorTable?.data?.length || 0),
 			CSSStyles: {
 				...styles.BODY_CSS,
 				'margin-top': '8px'
 			},
 			ariaLive: 'polite'
 		}).component();
 
+		this._loginSelectorTable = this._createNonSystemLoginTablesTable(view);
+
+		this._disposables.push(this._loginSelectorTable.onRowSelected(async (e) => {
+			await this.updateValuesOnSelection();
+		}));
+
+		// load unfiltered table list and pre-select list of logins saved in state
+		await this._filterTableList('', this.migrationStateModel._loginMigrationModel.loginsForMigration);
+
+		const flex = view.modelBuilder.flexContainer()
+			.withProps({ CSSStyles: { 'margin': '10px 0 0 15px' } })
```

Comment:
may be define CSS in styles file since it is being used multiple times.
https://github.com/microsoft/azuredatastudio/blob/c4af82dbfb68f738dc8d0357361bd8b13981bf1d/extensions/sql-migration/src/constants/styles.ts#L32